smart-contracts/contracts/interfaces/ILockCore.sol

```diff
@@ -85,4 +85,16 @@ interface ILockCore {
     external
     view
     returns (uint timestamp);
+
+  /**
+  * @param _page the page of key owners requested when faceted by page size
+  * @param _pageSize the number of Key Owners requested per page
+  */
+  function getOwnersByPage(
+    uint _page,
+    uint _pageSize
+  )
+    public
```

Comment:
minor issue, but we often get linter warnings about declaring functions as external in interfaces. I recommend changing this to external but keeping the public visibility in the implementation. Otherwise LG.